### PR TITLE
llvm: add 15.0.1, 15.0.2

### PR DESCRIPTION
**Pull request by prckent**

### CODE REVIEW

**Review by eugeneswalker — DISMISSED** [review]
Checksums verified. Thanks!

File: var/spack/repos/builtin/packages/llvm/package.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ class Llvm(CMakePackage, CudaPackage):
 
     # fmt: off
     version('main', branch='main')
+    version('15.0.2', sha256='dc11d35e60ab61792baa607dff080c993b39de23fb93b3d3369ba15b0601c307')
+    version('15.0.1', sha256='20bccb964e39f604fdc16d1258f94d2053fbdcdab2b2f6d5e20e6095ec403c00')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/llvm/package.py]: Style checks will (or have) failed due to the single quotes needing to be replaced with double.

**Review by tldahlgren — CHANGES_REQUESTED** [review]

**Review by tldahlgren — APPROVED** [review]
Also (re-) confirmed (all) sha256.

**Comment by tldahlgren** [comment]
@haampie @trws Did you want to comment on this PR before it is merged?

**Pull request merged**